trade-finance-manager-ui/server/controllers/utilisation-reports/find-reports-by-year/index.test.ts

```diff
@@ -341,7 +341,7 @@ describe('controllers/utilisation-reports/find-reports-by-year', () => {
         dateUploaded: '2024-02-15 10:38:01.4033333',
         totalFeesReported: 3,
         reportedFeesLeftToReconcile: 3,
-        formattedReportPeriod: 'January 2024',
+        formattedReportPeriod: 'January 2025',
```

Comment:
could we not maybe have a const on the top to get the year `new Date().getFullYear()` and then construct this string using the year const?

Reply:
Good shout, have added `formattedReportPeriod`. Thanks